_includes/_pagination.html

```diff
@@ -13,8 +13,19 @@
   <div class="load-more">
     <button class="btn {% if include.btn-type == 'white-outline' %}btn-white btn-outline{% else %}btn-blue{% endif %}" data-role="load-more" data-paginate="{{ include.collection }}">Load more{{ include.label | prepend: ' ' }}</button>
   </div>
-  {% endif %}
 
+  {% elsif include.n_of_total_pagination == "true" %}
+  <div class="text-center input-pagination">
+    <button class="btn btn-blue" id="pagination-previous-button"><<</button>
+    <input class="form-control form-control-custom-small" type="number"
+    max="{{paginator.total_pages}}" id="pagination-current-page" placeholder="{{paginator.page}}">
+    <span>of</span>
+    <span id="total_pages">{{paginator.total_pages}}</span>
+    <button class="btn" id="pagination-submit-button">Go!</button>
```

Comment:
With this button being grey it seems disabled even when it's not disabled. I think making it blue will help (I'll also have some changes on the prev/next)
```suggestion
    <button class="btn btn-blue" id="pagination-submit-button">Go!</button>
```